barretenberg/cpp/src/barretenberg/vm/avm_trace/avm_opcode.hpp

```diff
@@ -98,6 +98,8 @@ enum class OpCode : uint8_t {
     // Gadgets
     KECCAK,
     POSEIDON2,
+    SHA256,
+    PEDERSEN,
```

Comment:
It was a bug that these were missing since they're there in TS and Rust.

Reply:
You sir are a savior. Ty!